Makefile

```diff
@@ -182,7 +184,9 @@ $(O)/fs.part: $(O)/tools/mkfs $(FSEXTRA) $(UPROGS) intel-ucode/*
 $(O)/fs.img: $(O)/fs.part
 	dd if=$< of=$@ conv=sparse obs=512 seek=2048
 	truncate -s "51M" $@
-	parted -s --align optimal $@ mklabel gpt mkpart sv6filesystem 1MiB 50MiB
+	@if [ $(shell uname) != "Darwin" ]; then\
+		parted -s --align optimal $@ mklabel gpt mkpart sv6filesystem 1MiB 50MiB;\
+	fi # OSX does not have port of parted command
```

Comment:
This build step isn't going to work at all with this command skipped. Maybe add a TODO to address that?